Sources/FoundationInternationalization/TimeZone/TimeZone.swift

```diff
@@ -424,11 +287,11 @@ extension TimeZone : CustomStringConvertible, CustomDebugStringConvertible, Cust
     }
 
     public var description: String {
-        return "\(identifier) (\(_kindDescription))"
+        return _tz.debugDescription
```

Comment:
Should this be ?
```suggestion
        return _tz.description
```

Reply:
We only have the one implementation (`debugDescription`). I'll think about if we want to split them anyway.